packages/suite/src/config/wallet/networks.ts

```diff
@@ -12,7 +12,7 @@ const networks = {
             tx: 'https://btc1.trezor.io/tx/',
             account: 'https://btc1.trezor.io/xpub/',
         },
-        features: ['rbf', 'sign-verify'],
+        features: ['rbf', 'sign-verify', 'amount-unit'],
         customBackends: ['blockbook', 'electrum'],
         accountTypes: {
             taproot: {
```

Comment:
`amount-unit` features has to be added to `accountTypes.taproot.features` too for `btc`, `test` and `regtest`